samples/matter/weather_station/README.rst

```diff
@@ -0,0 +1,154 @@
+.. _matter_weather_station_sample:
+
+Matter: Weather station
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This weather station sample demonstrates the usage of the :ref:`Matter <ug_matter>` (formerly Project Connected Home over IP, Project CHIP) application layer to build a weather station device that lets you remotely gather different kind of data, such as like temperature, air pressure, and relative humidity, using the device sensors.
+This device works as a Matter accessory device, meaning it can be paired and controlled remotely over a Matter network built on top of a low-power, 802.15.4 Thread network.
+You can use this sample as a reference for creating your own application.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: thingy53_nrf5340
+
+To program and gather logs from the Thingy:53 device, you need the external J-Link programmer.
+If you own an nRF5340 DK that has an on-board J-Link programmer, you can also use it for this purpose.
+
+For remote testing scenarios, if you want to commission the lock device and :ref:`control it remotely <matter_lock_sample_network_mode>` through a Thread network, you also need the following:
+
+* A Matter controller device :ref:`configured on PC or smartphone <ug_matter_configuring>`.
+
+.. note::
+    |matter_gn_required_note|
+
+Overview
+********
+
+The sample uses buttons for changing the lock and device states, and LEDs to show the state of these changes.
+It can be tested in the following ways:
+
+* Standalone, by using a single DK that runs the door lock application.
+* Remotely over the Thread protocol, which requires more devices.
+
+The remote control testing requires a Matter controller, which can be configured either on PC or mobile (for remote testing in a network).
+Both methods can be enabled after :ref:`building and running the sample <matter_lock_sample_remote_control>`.
+
+.. _matter_weather_station_network_mode:
+
+Remote testing in a network
+===========================
+
+.. include:: ../lock/README.rst
+    :start-after: matter_door_lock_sample_remote_testing_start
+    :end-before: matter_door_lock_sample_remote_testing_end
+
+.. _matter_weather_station_test_mode:
+
+Remote testing using test mode
+==============================
+
+.. include:: ../lock/README.rst
+    :start-after: matter_door_lock_sample_test_mode_start
+    :end-before: matter_door_lock_sample_test_mode_end
+
+Configuration
+*************
+
+|config|
+
+Device Firmware Upgrade support
+===============================
+
+.. include:: ../lock/README.rst
+    :start-after: matter_door_lock_sample_build_with_dfu_start
+    :end-before: matter_door_lock_sample_build_with_dfu_end
+
+User interface
+**************
+
+.. include:: ../lock/README.rst
```

Comment:
I would remove it and add instead something like that:

> 
Button:
    Depending on how long you press the button:

    * If pressed for 6 seconds, it initiates the factory reset of the device.
      Releasing the button within the 6-second window cancels the factory reset procedure.
    * If pressed for less than 3 seconds, it starts the the NFC tag emulation, enables Bluetooth LE advertising for the predefined period of time, and makes the device discoverable over Bluetooth LE.This button is used during the :ref:`commissioning procedure <matter_lock_sample_remote_control_commissioning>`.